configure.ac

```diff
@@ -4272,6 +4284,13 @@ then
     fi
 fi
 
+if test "$ENABLED_BIND" = "yes"
+then
+    AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_BIND"
+    AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_AES_DIRECT -DHAVE_AES_ECB -DWOLFSSL_DES_ECB"
+    AM_CFLAGS="$AM_CFLAGS -DWOLFSSL_SHA224 -DWOLFSSL_SHA384 -DWOLFSSL_SHA512"
```

Comment:
Would be better if enabled_sha224="yes" and so on.
This way any other effects of having it enabled are also happening.